python/MooseDocs/extensions/MooseMarkdown.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, **kwargs):
     self.config['graphviz']     = ['/opt/moose/graphviz/bin', 'The location of graphviz executable for use with diagrams.']
     self.config['dot_ext']      = ['svg', "The graphviz/dot output file extension (default: svg)."]
     self.config['install']      = ['', "The location to install system and object documentation."]
+    self.config['macro_files']  = ['', "Comma deliminated paths to files that contain macros to be used in bibtex parsing."]
```

Comment:
This should also expect a python list, which you can provide in the moosedocs.yml as:

```
macro_files:
   - file1.bib
   - file2.bib
```